libraries.gradle

```diff
@@ -12,7 +12,7 @@ ext {
     junitVersion = '4.11'
 //    h2Version = '1.2.145'
     h2Version = '1.3.176'
-    bytemanVersion = '2.1.2'
+    bytemanVersion = '2.1.2's
```

Comment:
There is a typo here (the `s` char was probably added when you started searching for slf4j).